README.md

```diff
@@ -1,2120 +1,31 @@
-# Ember Data Factory Guy
+[![Build Status](https://github.com/adopted-ember-addons/ember-data-factory-guy/actions/workflows/ci.yml/badge.svg)](https://github.com/adopted-ember-addons/ember-data-factory-guy/actions/workflows/ci.yml) [![Ember Observer Score](http://emberobserver.com/badges/ember-data-factory-guy.svg)](http://emberobserver.com/addons/ember-data-factory-guy) [![npm version](https://badge.fury.io/js/ember-data-factory-guy.svg)](http://badge.fury.io/js/ember-data-factory-guy)
 
-[![Build Status](https://github.com/adopted-ember-addons/ember-data-factory-guy/workflows/ci/badge.svg)](https://github.com/adopted-ember-addons/ember-data-factory-guy/actions/workflows/ci.yml) [![Ember Observer Score](http://emberobserver.com/badges/ember-data-factory-guy.svg)](http://emberobserver.com/addons/ember-data-factory-guy) [![npm version](https://badge.fury.io/js/ember-data-factory-guy.svg)](http://badge.fury.io/js/ember-data-factory-guy)
+# Overview
 
 Feel the thrill and enjoyment of testing when using Factories instead of Fixtures.
 Factories simplify the process of testing, making you more efficient and your tests more readable.
 
-**NEW** starting with v3.8
-  - jquery is no longer required and fetch adapter is used with ember-data
-  - you can still use jquery if you want to  
-  - if you are addon author using factory guy set up your application adapter like [this](https://github.com/adopted-ember-addons/ember-data-factory-guy/blob/master/tests/dummy/app/adapters/application.js)
-  
-**NEW** starting with v3.2.1
-  - You can setup data AND links for your async relationship [Check it out](#special-tips-for-links)
+## Features
 
-**NEW** You can use factory guy in ember-twiddle
-  - Using [Scenarios](https://ember-twiddle.com/421f16ecc55b5d35783c243b8d99f2be?openFiles=tests.unit.model.user-test.js%2C)
+- 🚀 Easily push models to the Store, even the most complex relationships
+- ⚡️️ Simple, Powerful, & Intuitive API
+- ⭐ Works out of the box with your existing serializers and adapters
+- 💎 Multiple build strategies
+- 🕵️ Intercept and spy on Store requests
+- 🕐 Slow down response times
```

Comment:
Could we add links to all of the other pages until a time we get the site online into a public site? When we merge this PR the docs will only be available in this README

Reply:
@patocallaghan Sorry for the delay here :/ finally added a table of content to discover the documentation from the repo itself